src/app/core/routing/category/category-route.pipe.ts

```diff
@@ -7,6 +7,10 @@ import { generateCategoryUrl } from './category.route';
 @Pipe({ name: 'ishCategoryRoute', pure: true })
 export class CategoryRoutePipe implements PipeTransform {
   transform(category: CategoryView): string {
-    return generateCategoryUrl(category);
+    if (category) {
+      return generateCategoryUrl(category);
+    }
+
+    return;
```

Comment:
Why this additional check? This is done in the function already (and also done only in the function for product and page routes).